kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -71,6 +71,21 @@ namespace Testing
             rMassMatrix(0,0) = 1.0;
         }
 
+        void CalculateLumpedMassVector(
```

Comment:
I think this is the test that solves an ODE with the explicit strategy. If this is the case, the method that calculates the mass matrix shouldn't be no longer required and can be removed.

Reply:
Done in  d491e8a